src/vs/workbench/contrib/extensions/browser/extensionsViewlet.ts

```diff
@@ -617,6 +662,15 @@ export class ExtensionsViewPaneContainer extends ViewPaneContainer implements IE
 
 		this.notificationService.error(err);
 	}
+
+	private isSupportedDragElement(e: DragEvent): boolean {
+		if (e.dataTransfer) {
+			const typesLowerCase = e.dataTransfer.types.map(t => t.toLocaleLowerCase());
+			return typesLowerCase.indexOf('files') !== -1;
```

Comment:
We shall we allow only `.vsix` files here.

Reply:
Yes, that is something that I can look at.

Reply:
I have researched this and based on my findings the file information (ex: path, extension) is not available in the `dragenter` and `dragover` event handlers. As that information is not available, we are not able to inspect the files that are being dragged over. This information is available in the drop event handler so, we are able to filter out files that are not `.vsix` files.